closed/autoconf/custom-hook.m4

```diff
@@ -790,3 +791,70 @@ AC_DEFUN([OPENJ9_GENERATE_TOOL_WRAPPERS],
   OPENJ9_GENERATE_TOOL_WRAPPER([nasm], [$NASM])
   OPENJ9_GENERATE_TOOL_WRAPPER([rc], [$RC])
 ])
+
+AC_DEFUN([OPENJCEPLUS_SETUP],
+[
+  BUILD_JGSKIT=false
+  GSKIT_BIN=
+  GSKIT_SDK_BIN=
+  GSKIT_BIN_CREDENTIAL=
+  GSKIT_HOME=
+  OPENJCEPLUS_TOPDIR=
+
+  # check for GSKIT crypyo library for OpenJCEPlus
+  AC_ARG_WITH(gskit-bin, [AS_HELP_STRING([--with-gskit-bin], [specify path to the gskit bin url])])
+  AC_ARG_WITH(gskit-sdk-bin, [AS_HELP_STRING([--with-gskit-sdk-bin], [specify path to the gskit sdk bin url])])
+  AC_ARG_WITH(gskit-bin-credential, [AS_HELP_STRING([--with-gskit-bin-credential], [specify path to the gskit bin credential])])
+
+  AC_MSG_CHECKING([if gskit and gskit sdk bin url is set])
+  if test "x$with_gskit_bin" = xno || test "x$with_gskit_sdk_bin" = xno ; then
+    AC_MSG_RESULT([no])
+  elif test "x$with_gskit_bin" = x || test "x$with_gskit_bin" = xyes || test "x$with_gskit_sdk_bin" = x || test "x$with_gskit_sdk_bin" = xyes ; then
```

Comment:
Normally "" and "no" are given the same treatment. Other values (perhaps including "yes") should be taken as is.